app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def calculate_team_total_rating(teams: list) -> int:
+    result = [sum(team.get_rating() for team in teams)]
```

Comment:
Please specify annotation. List of what you expect

Simplify method and use correct naming for variable (player instead of team for example):
`return sum(player.get_rating() for player in teams)`